nextjs-app/app/components/ValueProposition.tsx

```diff
@@ -0,0 +1,280 @@
+
+              if (!imageUrl) {
+                return null
+              }
+
+              return (
+                <div key={index} className="flex-1 rounded-3xl overflow-hidden">
+                  <div className="relative w-full h-[300px] lg:h-[466px]">
+                    <Image
+                      src={imageUrl}
+                      alt={column.image.alt}
+                      fill
+                      className="object-cover"
+                      sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 624px"
+                    />
+                  </div>
+                </div>
+              )
+            }
+
+            if (column.columnType === 'testimonial') {
+              return (
+                <div key={index} className="flex-1">
+                  <InlineTestimonialCard
+                    quote={column.quote}
+                    authorName={column.authorName}
+                    authorTitle={column.authorTitle}
+                    authorImage={column.authorImage}
+                    backgroundColor={column.backgroundColor}
+                  />
+                </div>
+              )
+            }
+
+            const ButtonIcon = column.button?.icon
+              ? getLucideIcon(column.button.icon)
+              : null
+
+            return (
+              <div
+                key={index}
+                className="flex-1 rounded-3xl p-5 lg:p-12"
+                style={{ backgroundColor: column.backgroundColor }}
+              >
+                <h3 className="text-[40px] font-semibold text-gray-900 mb-8 leading-tight">
+                  {column.title}
+                </h3>
+
+                <div className="space-y-6 mb-8">
+                  {column.items.map((item, itemIndex) => {
+                    const ItemIcon = getLucideIcon(item.icon)
+
+                    return (
+                      <div key={itemIndex} className="flex items-start gap-4">
+                        <div className="flex-shrink-0 w-12 h-12 bg-white rounded-full flex items-center justify-center">
+                          <ItemIcon className="w-5 h-5 text-gray-700" />
+                        </div>
+
+                        <div className="flex-1 pt-1">
+                          <p className="text-lead text-[20px] text-gray-900 leading-relaxed">
+                            {item.text}
+                          </p>
+                        </div>
+                      </div>
+                    )
+                  })}
+                </div>
+
+                {column.button && column.button.label && column.button.url && (
+                  <div className="mt-8">
+                    <Link
+                      href={column.button.url}
+                      target="_blank"
+                      className="inline-block"
+                    >
+                      <Button iconPosition="right" variant="secondary">
+                        {ButtonIcon && <ButtonIcon className="mr-2 h-4 w-4" />}
```

Comment:
The existential checking behavior is treated differently here between `ItemIcon` and `ButtonIcon` here.

Is that intentional?  I dunno.. just seems like a code-smell since they're both calling `getLucideIcon` to get their respective icons. But one is a conditionally rendered, and the other is not? 🤔